README.md

```diff
@@ -3,6 +3,8 @@
 [![CircleCI](https://circleci.com/gh/mattermost/mattermost-plugin-solar-lottery.svg?style=shield)](https://circleci.com/gh/mattermost/mattermost-plugin-solar-lottery)
 [![Go Report Card](https://goreportcard.com/badge/github.com/mattermost/mattermost-plugin-solar-lottery)](https://goreportcard.com/report/github.com/mattermost/mattermost-plugin-solar-lottery)
 [![Code Coverage](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-solar-lottery/master.svg)](https://codecov.io/gh/mattermost/mattermost-plugin-solar-lottery)
+[![Release](https://img.shields.io/github/v/release/mattermost/mattermost-plugin-solar-lottery)](https://github.com/mattermost/mattermost-plugin-solar-lottery/releases/latest)
```

Comment:
this should be pre-release?

Reply:
Maybe `v0.1.0` can just be a normal release? We haven't been to consistent with using pre-releases?

Reply:
Copied from -> https://github.com/mattermost/mattermost-plugin-agenda/pull/41#discussion_r431947900

> the was a url query param that was available that includes pre-releases. They badge will be orange for found pre-releases and turn blue when the first release is done.

There is no pre-release for this plugin, so there badge will still show 
![image](https://user-images.githubusercontent.com/7575921/83165355-03c4e380-a0d3-11ea-8840-0931c64b7195.png)

Can we make a pre-release?

Reply:
Correction..  I just checked and with the new query param, we get the following which seems accurate and appropriate..
![image](https://user-images.githubusercontent.com/7575921/83165595-628a5d00-a0d3-11ea-8fc7-4ebb5e64f31b.png)

Reply:
Well, that is correct. There are no releases yet: https://github.com/mattermost/mattermost-plugin-solar-lottery/releases

Reply:
correct.  I was just pointing out that the latest commit seems the correct approach and can be added before any releases are added.  It will pickup pre-release and releases too.